packages/console/src/pages/Security/CreateCaptchaForm/constants.ts

```diff
@@ -5,6 +5,30 @@ import turnstile from '@/assets/images/turnstile.svg?react';
 
 import { type CaptchaProviderMetadata } from './types';
 
+const turnstileReadme = `
+# Cloudflare Turnstile
+
+Turnstile is a CAPTCHA service that helps protect your website from spam and abuse. This guide will walk you through the process of setting up Turnstile with Logto.
+
+## Prerequisites
+
+- A Cloudflare account
+
+## Setup
+
+1. Go to the [Cloudflare Dashboard](https://dash.cloudflare.com/login) and select your account.
+2. Navigate to **Turnstile** > **Add widget**.
+3. Fill out the form with the following details:
+   - **Widget name**: Any name you want to give to the widget
+   - **Hostname**: Logto's endpoint domain
```

Comment:
nit: Better give it an example URL here.

Reply:
updated